src/js/tracks/text-track-settings.js

```diff
@@ -241,14 +250,20 @@ function setSelectedOption(target, value) {
   target.selectedIndex = i;
 }
 
-function captionOptionsMenuTemplate() {
-  let template = `<div class="vjs-tracksettings">
-      <div class="vjs-tracksettings-colors">
-        <div class="vjs-fg-color vjs-tracksetting">
-            <label class="vjs-label">Foreground</label>
-            <select>
-              <option value="">---</option>
-              <option value="#FFF">White</option>
+function captionOptionsMenuTemplate(uniqueId, dialogLabelId, dialogDescriptionId) {
+
+  let template = `
+    <div role="document">
+      <h1 id="${dialogLabelId}" class="vjs-control-text">Captions Settings Dialog</h1>
```

Comment:
We probably should use a div here instead of h1, I had to change the elements used in videojs-dock because websites often style `h1` elements and it broke the title and descriptions of the plugin